pkgs/tools/misc/brltty/default.nix

```diff
@@ -13,10 +14,12 @@ stdenv.mkDerivation rec {
     sha256 = "14psxwlvgyi2fj1zh8rfykyjcjaya8xa7yg574bxd8y8n49n8hvb";
   };
 
-  nativeBuildInputs = [ pkg-config python3.pkgs.cython tcl ];
-  buildInputs = [ bluez ]
+  nativeBuildInputs = [ pkg-config python3.pkgs.cython tcl ]
+    ++ lib.optional espeak-ngSupport espeak-ng;
```

Comment:
What is the reason for adding this to `nativeBuildInputs` as well? It sounds like a purely runtime dependency and build seems to produce the same output without it here.